sympy/parsing/tests/test_c_parser.py

```diff
@@ -3038,18 +3550,10 @@ def test_var_decl():
         )
 
         c_src_raise1 = (
-            'double a;'
-        )
-
-        c_src_raise2 = (
-            'long a = 1;'
-        )
-
-        c_src_raise3 = (
             'int a = (int) 1.0;'
         )
 
-        c_src_raise4 = (
+        c_src_raise2 = (
             'int a = 10;'
             'int b = (a!=10)?(a):(0);'
         )
```

Comment:
These are giving error outputs on the tests, can you just remove these too,they're not being tested anyway? or maybe try some other tests like data types that have not been implemented yet, but the node visitors like transform_var_decl have been implemented?

Reply:
Yeah, it raises these errors
<pre>
====================================================================================== test process starts =======================================================================================
executable:         /home/smit/anaconda3/bin/python  (3.7.1-final-0) [CPython]
architecture:       64-bit
cache:              yes
ground types:       gmpy 2.0.8
numpy:              1.15.4
random seed:        87512398
hash randomization: on (PYTHONHASHSEED=3615327077)

sympy/parsing/tests/test_c_parser.py[13] .........Ignoring node of type CursorKind.CSTYLE_CAST_EXPR (( int ) 1.0)
Ignoring node of type CursorKind.CONDITIONAL_OPERATOR (( a != 10 ) ? ( a ) : ( 0 ))
....
</pre>

These nodes are unrecognizable, that's why! Alright I would remove these in the next commit...